libs/SalesforceSDKCore/SalesforceSDKCore/Classes/KeyValueEncryptedFileStore.swift

```diff
@@ -0,0 +1,395 @@
+//
```

Comment:
Could you put the file in a subdirectory?
E.g. `libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Storage` instead of `libs/SalesforceSDKCore/SalesforceSDKCore/Classes`